src/binaryen-shell.cpp

```diff
@@ -264,6 +265,7 @@ int main(int argc, const char* argv[]) {
   bool print_before = false;
   bool print_after = false;
   Name entry;
+  setDebugSignalHandler(argv[0]);
```

Comment:
All the programs now use support/command-line.cpp, so you should probably just set it from there.